dist/client.js

```diff
@@ -459,7 +459,6 @@
     define([], function() {
       return CrossStorageClient;
     });
-  } else {
-    root.CrossStorageClient = CrossStorageClient;
   }
+  root.CrossStorageClient = CrossStorageClient;
```

Comment:
I meant to say that the module loader is the one who imports the cross-storage script as well as many other scripts. In order to apply this technique I'd have to make the loader delete "module" selectively, depending on script it loads.